homeassistant/components/netatmo.py

```diff
@@ -50,7 +49,4 @@ def setup(hass, config):
         _LOGGER.error("Unable to connect to NatAtmo API")
         return False
 
-    for component in 'camera', 'sensor':
```

Comment:
Make it configurable but default it to `on` as automatic setup of platforms helps new users get started.